src/xcode/ENA/ENA/Source/SceneDelegate.swift

```diff
@@ -186,6 +187,8 @@ final class SceneDelegate: UIResponder, UIWindowSceneDelegate, RequiresAppDepend
 	}
 
 	private func presentHomeVC() {
+		Connectivity.urlSessionConfiguration.timeoutIntervalForRequest = 15.0
+		Connectivity.urlSessionConfiguration.timeoutIntervalForResource = 15.0
 		enStateHandler = ENStateHandler(
 			initialExposureManagerState: exposureManager.preconditions(),
 			reachabilityService: ConnectivityReachabilityService(
```

Comment:
In fact, the EnStateHandler has a strong reference to ConnectivityReachabilityService and meanwhile, ConnectivityReachabilityService also keeps the strong ref to EnStateHandler. please check it.